src/include/meta.h

```diff
@@ -34,6 +34,9 @@
 #define WT_HS_FILE "WiredTigerHS.wt"     /* History store table */
 #define WT_HS_URI "file:WiredTigerHS.wt" /* History store table URI */
 
+#define WT_CC_FILE "WiredTigerCC.wt"     /* Chunk cache table */
```

Comment:
Perhaps `WT_CC_METAFILE` and `WT_CC_METAFILE_URI` would be more readable?

Reply:
Fair call - I did it for consistency with the history store, but it's too easy to confuse with the fixed name for the chunk cache data file itself.

Fixed in 924ab47.